apps/miniapp/pages/index.tsx

```diff
@@ -1,89 +1,14 @@
-import { faker } from '@faker-js/faker'
-import { Typography } from 'antd'
-import dayjs from 'dayjs'
-import { get } from 'lodash'
-import getConfig from 'next/config'
 import Head from 'next/head'
-import { useState } from 'react'
+import React from 'react'
 
 import { useIntl } from '@open-condo/next/intl'
 
-import { Button } from '@condo/domains/common/components/Button'
-import { useLaunchParams } from '@condo/domains/miniapp/hooks/useLaunchParams'
 import { PageContent, PageHeader, PageWrapper } from '@miniapp/domains/common/components/BaseLayout'
-import { useOidcAuth } from '@miniapp/domains/common/utils/oidcAuth'
 
-const { publicRuntimeConfig: { condoUrl } } = getConfig()
-
-const TasksExample = () => {
-    const intl = useIntl()
-    const TasksTitleMsg = intl.formatMessage({ id: 'pages.index.tasks.Title' })
-    const TasksAddMsg = intl.formatMessage({ id: 'pages.index.tasks.Add' })
-    const TasksUpdateMsg = intl.formatMessage({ id: 'pages.index.tasks.Update' })
-
-    const [latestTaskRecord, setLatestTaskRecord] = useState({})
-
-    const postMessageToAddTask = () => {
-        // Should correspond to TaskRecord type
-        const record = {
-            id: faker.datatype.uuid(),
-            status: 'processing',
-            title: 'Task from mini-app',
-            description: `Created at ${dayjs().format('YYYY-MM-DD HH:mm:ss')}`,
-            progress: 50,
-            // Used to find appropriate `ITask` interface implementation for this record
-            __typename: 'MiniAppTask',
-        }
-        const message = {
-            type: 'task',
-            command: 'create',
-            record,
-        }
-        setLatestTaskRecord(record)
-        window.postMessage(message, condoUrl)
-    }
-
-    const postMessageToUpdateTask = () => {
-        const record = {
-            ...latestTaskRecord,
-            description: `Updated at ${dayjs().format('YYYY-MM-DD HH:mm:ss')}`,
-            progress: Math.floor(Math.random() * 100),
-        }
-        const message = {
-            type: 'task',
-            command: 'update',
-            record,
-        }
-        window.postMessage(message, condoUrl)
-    }
-
-    return (
-        <>
-            <Typography.Title level={2}>
-                {TasksTitleMsg}
-            </Typography.Title>
-            <Button onClick={postMessageToAddTask} type='primary'>
-                {TasksAddMsg}
-            </Button>
-            <Typography.Paragraph>
-                Latest added task: {JSON.stringify(latestTaskRecord, null, '\t')}
-            </Typography.Paragraph>
-
-            <Button onClick={postMessageToUpdateTask} type='primary'>
-                {TasksUpdateMsg}
-            </Button>
-        </>
-    )
-}
-
-const IndexPage = () => {
+const IndexPage: React.FC = () => {
     const intl = useIntl()
-    const auth = useOidcAuth()
```

Comment:
Why you want to drop all oidc miniapp code?

Reply:
I'm not dropping it, just not using user info at this page. Btw, I would kill all miniapp code, since it's a legacy mess right now :)